### PR TITLE
Fix/revert to readcounter

### PR DESCRIPTION
**Pull request by lpoli**
###Changes
This PR reverts Readmarker changes from read size to read counter

Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/zboxcli/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

Associated PRs (Link as appropriate):
- 0chain: https://github.com/0chain/0chain/pull/1146
- blobber: https://github.com/0chain/blobber/pull/632
- gosdk: https://github.com/0chain/gosdk/pull/429
- system_test:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -59,4 +59,4 @@ require (
 )
 
 // temporary, for development
-//replace github.com/0chain/gosdk => ../gosdk
+replace github.com/0chain/gosdk => ../gosdk
```
Comment (peterlimg) [inline go.mod]: This go mod replace should not be committed

**Review by peterlimg — CHANGES_REQUESTED** [review]

**Review by peterlimg — APPROVED** [review]
lgtm

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | fix/revert-to-readcounter  |
| zwalletcli  | staging  |
| 0chain  | fix/revert-to-readcounter  |
| blobber  | fix/revert-to-readcounter  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | fix/revert-to-readcounter  |
| zboxcli  | fix/revert-to-readcounter  |
| zwalletcli  | staging  |
| 0chain  | fix/revert-to-readcounter  |
| blobber  | fix/revert-to-readcounter  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/revert-to-readcounter  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | fix/revert-to-readcounter  |
| zwalletcli  | staging  |
| 0chain  | fix/revert-to-readcounter  |
| blobber  | fix/revert-to-readcounter  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/revert-to-readcounter  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | fix/revert-to-readcounter  |
| zwalletcli  | staging  |
| 0chain  | fix/revert-to-readcounter  |
| blobber  | fix/revert-to-readcounter  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Pull request merged**